MessageViewController/MessageView.swift

```diff
@@ -110,50 +110,30 @@ public final class MessageView: UIView, MessageTextViewListener {
         }
     }
 
-    public var buttonLeftInset: CGFloat = 0 {
+    public var leftButtonInset: CGFloat = 0 {
         didSet { setNeedsLayout() }
     }
 
     public func set(buttonIcon: UIImage?, for state: UIControlState, type: buttonType) {
         switch type {
-            case .left:
-                setLeft(buttonIcon: buttonIcon, for: state)
-                break
-            case .right:
-                setRight(buttonIcon: buttonIcon, for: state)
-                break
+        case .left:
```

Comment:
We could even make the `switch` only get the button:

```swift
let button: UIButton
switch type {
case .left:
  button = leftButton
case .right
  button = rightButton
}
button.setImage(buttonIcon, for: state)
```

Reply:
Good idea! I have implemented that 👍